docs/data/data-grid/components/CustomFooter.tsx

```diff
@@ -31,7 +31,7 @@ export default function CustomFooter() {
   });
   return (
     <Box sx={{ width: 1 }}>
-      <Box sx={{ height: 350, width: 1, mb: 2 }}>
+      <Box sx={{ height: 350, width: '100%', mb: 2 }}>
```

Comment:
To keep consistent with https://github.com/mui/mui-x/pull/5177/files#diff-20c5e5917be2dde0d2608131f875475a1c54484b869ba10d56786cc1728b15f7R101, it should be 1 unit, or 2 doing the inverse.

```suggestion
      <Box sx={{ height: 350, width: '100%', mb: 1 }}>
```